src/main/java/ca/ciccc/Assignment2.java

```diff
@@ -90,9 +152,18 @@ public boolean tripleIncreasingOrder(int[] nums) {
      * @return true if the array contains the same number of odds and even numbers, otherwise false.
      */
     public boolean evenOrOdd(int[] nums){
-        // Todo: your code goes here
+        int evens = 0;
+        int odds = 0;
+        
+        for (int i = 0; i < nums.length; ++i) {
+            if (nums[i] % 2 == 0) {
+                evens += 1;
+            } else {
+                odds += 1;
+            }
+        }
 
-        return false;
+        return evens == odds;
```

Comment:
You have to return true or false depending if is even or odds.

Reply:
@Enrique92 Thanks for a comment! I think `evens == odds` will be a boolean value. So it's fine with only returning it.